incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,19 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates all objects of a [given type](#admin-hc) from ancestor namespaces to descendant namespaces. However, sometimes this is too restrictive, and you need to create exceptions to certain policies. For example:
```

Comment:
Can you please wrap all these lines to similar lengths as the rest of this doc? I _think_ it's 80 characters but it might be 100. There should be an option in your editor to do this.